RestApiHomeTask/Controllers/ItemController.cs

```diff
@@ -0,0 +1,120 @@
+using Microsoft.AspNetCore.Mvc;
+using Npgsql;
+using RestApiHomeTask;
+
+namespace RestApiHomeTask.Controllers;
+
+[ApiController]
+[Route("api/[controller]/[action]")]
```

Comment:
same about route names as in categories controller

Reply:
I have update last commit this changes